fishtank.py

```diff
@@ -312,7 +125,7 @@ def main():
     print("-")
     print("Average age of Ogres at death: %s" % str(current_room.ogre_average_death_age))
     print("Average number of Goblins eaten: %s" % str(current_room.ogre_average_goblins_eaten))
-
     pygame.quit()
 
-main()
+if __name__ == "__main__":
+    main()
```

Comment:
This if-statement lets you know whether the file is being run, or just imported. We only want to run main() if the file is being executed. (Kind of a pedantic fix)